kubetest/e2e.go

```diff
@@ -732,6 +717,20 @@ func skewTestEnv(env, args []string, prefix string, checkSkew bool) error {
 	return control.FinishRunning(cmd)
 }
 
-func test(testArgs []string) error {
+// GinkgoScriptTester implements Tester by calling the hack/ginkgo-e2e.sh script
+type GinkgoScriptTester struct {
```

Comment:
should this be in the e2e package?

Reply:
I was trying to keep the diff simpler.  Happy to move it if you prefer, but my hope is that this goes away...